dev/docker/docker-compose.yml

```diff
@@ -50,8 +50,7 @@ services:
         - ./phpdocker/php-fpm/php-ini-overrides.ini:/etc/php/7.2/fpm/conf.d/99-overrides.ini
```

Comment:
Increment `image` to `tribe-phpfpm:7.2-rev1` to force a rebuild for other users when you change the `Dockerfile`.